packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -101,6 +102,10 @@ const useLiveChat = (has_cookie_account = false) => {
         });
     };
 
+    useEffect(() => {
+        onHistoryChange();
+    }, [search_params, onHistoryChange]);
```

Comment:
i dont think useEffect will run on `window.location.search` since this is managed externally, outside of react. Can you replace with things in `react-router`?

Reply:
@yashim-deriv I can change, no problem, but I've tested it and it works with `window.location.search`.

Reply:
@yashim-deriv Changed to useParams(), pls check when you have time